Makefile

```diff
@@ -0,0 +1,5 @@
+pdf:
```

Comment:
I think make builds the first target by default if you don't have all, but I'll add it.

I need to add that stop on error command back in too.